src/program/lwaftr/virt/lwaftrctl

```diff
@@ -49,16 +48,17 @@ function qemu_start_vm {
         "sudo numactl -m ${NUMA_NODE} taskset -c ${QEMU_CORE} \
             $QEMU $QEMU_ARGS \
```

Comment:
Here QEMU_ARGS will be unset. Still works but the intention to define QEMU_ARGS was to follow snabb-nfv setup and to accommodate a variable in case someone wants to pass arguments to QEMU.

See for instance how it's used in snabb-nfv/test-env. It's assigned a value in case an initrd asset is defined:

https://github.com/Igalia/snabb/blob/lwaftr/src/program/snabbnfv/test_env/test_env.sh#L116

I think you needed to do something with initrd. Perhaps is worth keeping QEMU_ARGS.